tools/gm-to-obj/README.md

```diff
@@ -0,0 +1,16 @@
+## .GM to .OBJ file converter
+
+### Requirements
+In order to run this script you need:
+- Python 3+
+- Some .gm files from one game
```

Comment:
I have a problem converting RESOURCE/MODELS/light.gm. Please make sure that all gm files from Sea Dogs: To Each His Own can be successfully converted.

Reply:
I'm going to investigate why. GM Viewer can't open that file either...

Reply:
I would not be surprised, if there are a lot of gm files that are not used at all. Maybe some leftovers from the previous games. Maybe even the gm file format has changed since then. 
But we have to find it out to make the transition from gm to obj as smooth as possible.

Reply:
Sure, I've also seen that a couple of ships crash the script as well, so there is surely a bug in my code :) 
I'm going to have a deeper look when I got some time